core/ppu.hpp

```diff
@@ -22,17 +22,103 @@ class PPU {
   // have to update the buffer when SDL asks for it
   u8* getFrameBuffer();
 
-  // probably a lot of other public interface stuff that you'll discover in implementation
-  // See https://gbdev.io/pandocs/Rendering.html
+  // TODO: decide whether this logic should live in the PPU or if we should have the MMU handle 
```

Comment:
The more I've thought about it, the more that I think I prefer having the PPU's mode be accessible by the MMU. Would that work?